apis/index.js

```diff
@@ -103,9 +101,7 @@ export async function updateLocation(regionKey, key, newLocation) {
   try {
     const existing = await fetchLocation(key);
     if (existing && existing.key && existing.updated >= newLocation.updated) {
-      Alert.alert(i18n.t('location/errorUpdate'), i18n.t('location/errorUpdateDescription'), [
-        { text: i18n.t('button/ok'), onPress: () => {} },
-      ]);
+      showOutdatedAlert();
```

Comment:
Dross. Maybe we need to return whether it was not changed or not then have the caller show the alert. Maybe that is too much.